### PR TITLE
fix for incorrect page redirect from Landing to info collection

### PR DESCRIPTION
**Pull request by aswin1689**
### What was the feature/problem?

https://github.com/homedepot/infinite-wish-board/issues/115

### How does this PR address the above feature/problem?

User is not navigated to child information collection page until user has entered a name, age and then click on a wish type.

### Check lists (check `x` in `[ ]` of list items)

- [x] Test passed
- [x] Coding style (indentation, etc)

### Additional Comments (if any)
Only redirecting the user to next page after entering a name, age and then click on wish type for now since there is no next button on this page.

### CODE REVIEW

**Comment by WRMilling** [comment]
Hey @aswin1689 If you don't mind resolving the conflicts with master, I will go ahead and see about merging.

**Comment by aswin1689** [comment]
@WRMilling The conflicts are resolved now. Thanks

File: ui/src/landing/CreateWish.js [hunk 1]
```diff
@@ -3,40 +3,52 @@ import { Landing } from './splashScreen'
...
     this.setState({ [field]: value })
   }
 
-  selectWishType = (wishType) => {
-    this.setState({ wishType })
+  selectWishType = wishType => {
+    const { name, age } = this.state
+
+    if (name !== '' && age !== '') {
+      this.setState({ wishType, showChildInfo: true })
```
Comment (WRMilling) [inline ui/src/landing/CreateWish.js]: Looks like this path is missed in the tests according to CodeCov, potentially modifying the test to use `selectWishType` directly vs. modifying the `showChildInfo` variable should resolve it. 

Let me know what you think.

File: ui/src/landing/CreateWish.test.js [hunk 2]
```diff
@@ -32,18 +32,26 @@ describe('CreateWish tests', () => {
...
     })
 
+    it('Should hide Childinfo component', () => {
+      expect(wrapper.find(Childinfo).length).toEqual(0)
+    })
+
     it('Should show Childinfo component', () => {
+      wrapper.setState({
+        showChildInfo: true
+      })
```
Comment (WRMilling) [inline ui/src/landing/CreateWish.test.js]: Something like `wrapper.instance().selectWishType("GO")` should resolve the codecov miss mentioned in `CreateWish.js`.

File: ui/src/landing/CreateWish.js (hunk 1 above)
Comment (aswin1689) [reply ui/src/landing/CreateWish.js]: Thanks. I have made the change and pushed.

**Comment by WRMilling** [comment]
Thank you for the updates @aswin1689.

**Pull request merged**